packages/flutter_localizations/lib/src/l10n/cupertino_mr.arb

```diff
@@ -0,0 +1,22 @@
+{
+  "todayLabel": "a temporary hack",
```

Comment:
the latin script in this file is a placeholder while the remaining 3 strings are being translated